Android.common.mk

```diff
@@ -111,12 +111,14 @@ LOCAL_CPPFLAGS += \
 	-DUSE_GL
 endif
 
+ifeq ($(strip $(BOARD_USES_LIBVA)), true)
```

Comment:
We want to change this and have something like DISABLE_MEDIA_COMPSITOR and only your target would enable this config.